test_project/store/src/main/java/test/sergeyvalyushko/store/handlers/TopCatalog.java

```diff
@@ -6,25 +6,22 @@
 
 import javax.xml.parsers.ParserConfigurationException;
 import java.io.IOException;
-import java.util.ArrayList;
 import java.util.List;
 
-public class TopCatalog extends AbstractHandler {
+public class TopCatalog extends SortCatalog {
+
     public TopCatalog(AbstractHandler nextHandler) {
         super(nextHandler);
+        sortParameters.add("top");
```

Comment:
not valid statement, cause in this handler you will have also name and price consts, and if this handler will be inited befre sort handler, only this one will handle all requests.